src/components/Game/Game.js

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 
-import { sample } from '../../utils';
-import { WORDS } from '../../data';
+import { sample } from "../../utils";
+import { WORDS } from "../../data";
+import Guess from "../Guess/Guess";
+import Grid from "../Grid";
+import Banner from "../Banner";
 
 // Pick a random word on every pageload.
 const answer = sample(WORDS);
 // To make debugging easier, we'll log the solution in the console.
 console.info({ answer });
 
 function Game() {
-  return <>Put a game here!</>;
+  const [guessHistory, setGuessHistory] = useState([]);
+
+  const handleSubmitGuess = (guess) => {
+    setGuessHistory((prevGuessHistory) => [...prevGuessHistory, guess]);
+  };
+
+  const mostRecentGuess = guessHistory.at(-1);
+  const numGuesses = guessHistory.length;
+  const isCorrectGuess = mostRecentGuess === answer;
+  const gameOver = numGuesses > 5 || isCorrectGuess;
+
+  return (
+    <>
+      <Grid guessHistory={guessHistory} correctAnswer={answer} />
+      <Guess handleSubmitGuess={handleSubmitGuess} gameOver={gameOver} />
+      {gameOver ? (
+        <Banner
+          wonGame={isCorrectGuess}
+          numGuesses={numGuesses}
+          answer={answer}
+        />
+      ) : (
+        ""
+      )}
```

Comment:
i believe you can use `&&` here because if it computes to false it doesn't get rendered. 

```suggestion
      {gameOver && (
        <Banner
          wonGame={isCorrectGuess}
          numGuesses={numGuesses}
          answer={answer}
        />
      )}
```

Reply:
Hm so do you test it out? And if using `&&` doesn't break anything and results in cleaner code then you use it over a ternary?

Reply:
The reviewer basically said yes to what I said above but in person because he doesn't want to leave a paper trail of this scandalous gravely incriminating evidence of treachery.